pkg/runner/cluster_k8s.go

```diff
@@ -293,138 +293,60 @@ func (*ClusterK8sRunner) ID() string {
 	return "cluster:k8s"
 }
 
-func (c *ClusterK8sRunner) healthcheckK8s() (k8sCheck api.HealthcheckItem) {
-	k8sCheck = api.HealthcheckItem{Name: "k8s", Status: api.HealthcheckStatusOK, Message: "k8s cluster is running"}
-	cmd := exec.Command("kops", "validate", "cluster")
```

Comment:
I like that we are removing this - it will be a pain (and most probably a mistake) to move `kops validate cluster` to k8s, once the daemon is on the remote cluster.